streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -580,6 +582,23 @@ public void setGlobalStateRestoreListener(final StateRestoreListener globalState
         }
     }
 
+    /**
+     * Set the listener which is triggered whenever a standby task is updated
+     *
+     * @param standbyListener The listener triggered when a standby task is updated.
+     * @throws IllegalStateException if this {@code KafkaStreams} instance has already been started.
+     */
+    public void setStandbyUpdateListener(final StandbyUpdateListener standbyListener) {
```

Comment:
IMO, in the context of this method, `standbyListener` makes more sense than `userStandbyListener`.

Reply:
Agreed!